app/src/scene/camera/camera.cpp

```diff
@@ -50,10 +59,72 @@ const glm::mat4 &Scene::Camera::view() const
     return _view;
 }
 
+void Scene::Camera::handleEvent(Event *event)
+{
+    switch (event->type())
+    {
+    case Event::EventType::KEY_PRESS:
+        keyPressEvent(static_cast<KeyPressEvent *>(event));
+        return;
+    case Event::EventType::KEY_RELEASE:
+        keyReleaseEvent(static_cast<KeyReleaseEvent *>(event));
+        return;
+    case Event::EventType::MOUSE_MOVE:
+        mouseMoveEvent(static_cast<MouseMoveEvent *>(event));
+        return;
+    default:
```

Comment:
please describe  default case here, we must have default behavior

Reply:
We don't need the default behavior cause the class of Camera is the last of the events handlers